### DISCUSSION TITLE
API Problem with assigning S3 Storage to Radio Station

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#4639] [General]
Hi,

I have a problem assigning a S3 Bucket to a Radio Station, I am using the following JSON,
{
"stations": [
    "Station Name"
  ]
}

I am using the PUT command to accomplish this.
After running the JSON code it comes back "successful" but when checking the Radio Station ID with GET, it has not assigned the  S3 Bucket/path to the Station.

Kind Regards,

**Comment by ghost** [comment]
Anybody?

**Reply by ghost** [reply]
Still nobody? Anybody!

**Comment by BusterNeece** [comment]
You should be setting `media_storage_location_id` on the station itself.

Also, please don't post follow-up messages like that. We get e-mails with every single message posted here, and it isn't respectful of our limited free time and resources to demand that we respond in a certain timeframe.

**Comment by ghost** [comment]
what ever man, you are very disrespectful towards your users and you
act like a baby who lost its toys, GROW UP DUDE!!!!!!!!!!
Message like those will not be posted if you answer questions politely and on time, I can see how you ignore people on the forums, it looks like C%$#P, fix it or stop it!

**Reply by ItsMitchh** [reply]
Your lack of empathy and regard for our team is astounding, consider yourself restricted from this repo and we will not provide any support or entertain your questions. 

I hope you can find some respect and empathy elsewhere, kind regards
